### DISCUSSION TITLE
sound effects

### DISCUSSION THREAD

**Discussion by odaymusic** [naomiaro/waveform-playlist#167] [General]
Can I use it with next js and how is the correct way to add sound effects .. I am trying to find a good explanation for adding effects

**Comment by naomiaro** [comment]
Sound effects currently aren't supported yet: https://github.com/naomiaro/waveform-playlist/issues/24

**Comment by naomiaro** [comment]
For next js you'd have to create a React wrapper component like you would do for a library you wanted to use that isn't written in React. https://reactjs.org/docs/integrating-with-other-libraries.html

**Comment by odaymusic** [comment]
Can I pay money for adding sound effects?

**Reply by odaymusic** [reply]
؟؟

**Reply by naomiaro** [reply]
I'm looking into opening a targeted sponsorship tier with github sponsors to maybe undertake this in 2022. Is there something specific you had in mind?

**Reply by odaymusic** [reply]
Of course, I will support you financially and morally. I look forward to your support in developing some tools such as pizzicato js, ​​where each track contains a rac that carries the effects you choose and changes its amount in the slider, as shown in pizzicato. It also supports playing midi files and it would be a good thing if there was a search engine that pulls audio files from websites and banks The sound is for the creators to use with the editor and set functions to suit playback and sizes on the mobile, as JavaScript is important, but everyone now uses the mobile phone and needs a tool like this on the phone.
